app/src/components/common/spinner/index.tsx

```diff
@@ -34,14 +35,15 @@ interface Props extends HTMLAttributes<HTMLDivElement> {
   color?: string
   height?: string | undefined
   width?: string | undefined
+  big?: boolean | undefined
```

Comment:
@mtsalenc , could you change `big` to `large` ?

Reply:
Done.